### PR TITLE
render new exhibition guide types

### PR DESCRIPTION
**Pull request by gestchild**
Relates to #10820

### CODE REVIEW

File: content/webapp/components/ExhibitionGuideLinks/ExhibitionGuideLinks.tsx [hunk 1]
```diff
@@ -112,4 +97,61 @@ const ExhibitionGuideLinks: FunctionComponent<Props> = ({
   );
 };
 
-export default ExhibitionGuideLinks;
+type ResourceProps = {
+  textPathname: string | undefined; // TODO make these more specifid=c?
```
Comment (rcantin-w) [inline content/webapp/components/ExhibitionGuideLinks/ExhibitionGuideLinks.tsx:101]: Is still still todo? 

And do we have a convention/preference on

`textPathname?: string`
vs
`textPathname: string | undefined`?

File: content/webapp/components/GuideMediaStop/GuideMediaStop.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+} from '@weco/content/components/Body/GridFactory';
+
+export type Props = {
+  // TODO
+  number: any;
+  title: any;
+  audio: any;
+  video: any;
+  type: any;
+};
```
Comment (rcantin-w) [inline content/webapp/components/GuideMediaStop/GuideMediaStop.tsx:21]: Should this be addressed?

File: content/webapp/pages/guides/exhibitions/[id]/[type].tsx [hunk 3]
```diff
@@ -266,14 +359,36 @@ const ExhibitionGuidePage: FunctionComponent<Props> = props => {
...
-                <p>This exhibition has {numberedStops.length} stops.</p>
+                <p>This exhibition has {numberOfStops} stops.</p>
               )}
             </>
           )}
         </Layout>
       </Space>
-
-      <ExhibitionGuideStops type={type} stops={exhibitionGuide.components} />
+      {/* For deprecated ExhibitionGuides */}
```
Comment (rcantin-w) [inline content/webapp/pages/guides/exhibitions/[id]/[type].tsx:368]: Are we thinking of migrating those to the new ones/deleting them eventually?

**Review by rcantin-w — COMMENTED** [review]
Would it be nice/doable to split legacy EG and new EG renderings in the `exhibitions/[id]/index` file?

File: prismic-model/src/exhibition-highlight-tours.ts [hunk 4]
```diff
@@ -0,0 +1,47 @@
+// import { CustomType } from './types/CustomType';
+import title from './parts/title';
+import { documentLink, mediaLink } from './parts/link';
+import list from './parts/list';
+// import image from './parts/image'; // TODO do we need this?
```
Comment (rcantin-w) [inline prismic-model/src/exhibition-highlight-tours.ts:5]: A few TODOs in this file to be removed?

File: prismic-model/src/exhibition-texts.ts [hunk 5]
```diff
@@ -0,0 +1,44 @@
+import { CustomType } from './types/CustomType';
+import title from './parts/title';
+import { documentLink } from './parts/link';
+import list from './parts/list';
+// import image from './parts/image'; // TODO is this needed?
```
Comment (rcantin-w) [inline prismic-model/src/exhibition-texts.ts:5]: Same here?

**Review by rcantin-w — APPROVED** [review]
Ran it locally and legacy EG still looks perfect (I think you said that was the way to test it?) 👍

File: content/webapp/pages/guides/exhibitions/[id]/[type].tsx (hunk 3 above)
Comment (gestchild) [reply content/webapp/pages/guides/exhibitions/[id]/[type].tsx:368]: I'm not sure, I think it's more likely that the get deleted than migrated - I'll raise it with the content team

**Pull request merged**